app/src/main/java/org/fossasia/phimpme/gallery/SelectAlbumBottomSheet.java

```diff
@@ -128,9 +129,24 @@ public void onClick(View view) {
 		builder.setPositiveButton(R.string.ok_action, new DialogInterface.OnClickListener() {
 		  @Override
 		  public void onClick(DialogInterface dialogInterface, int i) {
+              int folderCount=folders.size();
+              int check=1;
+              int filePosition=0;
+              while (filePosition<folderCount) {
+                  File f = folders.get(filePosition);
+                  filePosition++;
+                  if (editText.getText().toString().equals(f.getName()))
+                      check=0;
+              }
+              if(!editText.getText().toString().trim().isEmpty() && check!=0) {
 			File folderPath = new File(currentFolderPath.getText().toString() + File.separator + editText.getText().toString());
 			if (folderPath.mkdir()) displayContentFolder(folderPath);
+              }
+              else if(check==0)
+                  Toast.makeText(getContext(),R.string.folder_name_exists,Toast.LENGTH_SHORT).show();
```

Comment:
Use a Snackbar instead

Reply:
@anantprsd5 I think since this is a dialog a toast message would be better because when I use a snackbar it is not visible till I move the dialog to full screen. What do you think?
![ezgif com-video-to-gif 10](https://user-images.githubusercontent.com/20878145/31028648-550d573c-a56c-11e7-8810-2305f4804c86.gif)